### PR TITLE
Version bump to fix SECURITY-3430 / CVE-2024-43044

### LINKED ISSUE #1827: Upgrade to fix CRITICAL SECURITY-3430 / CVE-2024-43044
The current versions in any tags are affected by  https://www.jenkins.io/security/advisory/2024-08-07/#SECURITY-3430

As per link before, the Jenkins version should be bumped to Jenkins 2.471, LTS 2.452.4, LTS 2.462.1

### PR DESCRIPTION
**Pull request by AlbertoPimpo**
Fixes #1827

### CODE REVIEW

**Comment by sayan-biswas** [comment]
This can be included in this PR as well.
https://github.com/openshift/jenkins/pull/1758#issuecomment-2275205514

File: 2/contrib/openshift/jenkins-version.txt [hunk 1]
```diff
@@ -1 +1 @@
-2.440.3
+2.452.4
```
Comment (sayan-biswas) [inline 2/contrib/openshift/jenkins-version.txt:1]: LST version 2.462.1 also has the fix.
https://www.jenkins.io/security/advisory/2024-08-07/

Comment (AlbertoPimpo) [reply 2/contrib/openshift/jenkins-version.txt]: Ok, changed to 2.462.1

**Comment by AlbertoPimpo** [comment]
Moving this change to https://github.com/openshift/jenkins/pull/1758

**Pull request closed without merging**